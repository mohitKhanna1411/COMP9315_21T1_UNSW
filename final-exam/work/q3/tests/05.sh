./bs 32 + -31 -0 -30 -1
