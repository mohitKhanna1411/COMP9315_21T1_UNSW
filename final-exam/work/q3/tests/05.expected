init: 11111111111111111111111111111111
-031: 01111111111111111111111111111111
-000: 01111111111111111111111111111110
-030: 00111111111111111111111111111110
-001: 00111111111111111111111111111100
